classes/class-wc-connect-api-client.php

```diff
@@ -193,6 +193,7 @@ protected function request( $method, $path, $body = array() ) {
 				'last_heartbeat' => get_option( 'wc_connect_last_heartbeat', 0 ),
 				'last_rate_request' => get_option( 'wc_connect_last_rate_request', 0 ),
 				'active_services' => $this->wc_connect_loader->get_active_services(),
+				'enable_stats' => ! Jetpack::is_staging_site(),
```

Comment:
Hum... I agree on the server logic part, but I've never been a fan of "inverse booleans" such as ```var doesntHaveSomething``` or ```somethingIsDisabled```, it makes the code harder to read.

What do you mean by filter?